tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -407,7 +407,7 @@ private static void InitializeServerless(Stopwatch sw, LambdaMetadata metadata)
         {
             try
             {
-                Serverless.InitIfNeeded(metadata);
+                // Serverless.InitIfNeeded(metadata);
```

Comment:
⚠️

Reply:
I would have say you can remove all this code in this PR, but doing in a follow up PR is fine if you prefer 👍

Reply:
updated. Now it's all cleaned up.